.github/workflows/smoke-test.yml

```diff
@@ -18,10 +18,10 @@ jobs:
   smoke_test:
     strategy:
       matrix:
-        test-script:
-          [
+        test-script: [
             ./smoke-test.bash,
-            ./smoke-test-l3.bash,
+            # TODO: Fix later
```

Comment:
Created a backlog ticket for this: https://app.asana.com/1/1208976916964769/project/1209446932824128/task/1210035248212787